### DISCUSSION TITLE
PhotoPrism Folder View

### DISCUSSION THREAD

**Discussion by Tailsproxy** [Radiokot/photoprism-android-client#19] [Ideas]
Finally something akin to Nextclouds various gallery apps. This is already running stably, super fast and responsive, at least on my end.  Really excited to see how this develops!  

Not sure how feasible this is but one feature that would increase the apps utility would be an ability to view PhotoPrism folders.

**Comment by Radiokot** [comment]
Thanks for checking out the app, I'm glad you found it useful.
You can browse folders content in the app using search filters and bookmarks. You can use "folder:" filter or just enter the folder name, as PhotoPrism adds folder names as keywords to the media. For example, here is how I browse my screenshots:
<p float="left">
<img src="https://user-images.githubusercontent.com/5675681/228633071-0e907e42-7eea-40de-94ce-53050e1774c3.png" width=300 />
<img src="https://user-images.githubusercontent.com/5675681/228633100-d66934c6-499b-4a1a-81ad-28afb6776e5b.png" width=300 />
</p>

My first priority is to implement features missing in the official web app. Copying the existing features is not as necessary, since the web app works pretty well.

**Comment by Radiokot** [comment]
Here is the guide on all the filters:
https://docs.photoprism.app/user-guide/search/filters/
PhotoPrism search is very powerful!

**Comment by Tailsproxy** [comment]
Thanks for clearing that up, I didnt realise how powerful  the search was and understand why its not a priority when you can get much of the function already.

I have been regularly checking F-Droid for updates the last few days and just saw you updated the app again!

My app is now matches my system theme and looks great. I also noticed the little lightbulb icon which directs users to the search functions you mentioned, very handy to have easy access to that.

**Comment by Radiokot** [comment]
Hey, check out the latest release – it has filtering by albums and folders ;)

**Reply by Tailsproxy** [reply]
Yes I saw that, that is very useful, thank you!

Also, congratulations on the new job, I hope it is going well and you have settled in. It's good to see you are still finding time to work on the app despite the new role :)

**Reply by Radiokot** [reply]
Thank you 😃
Things on the new job are going well, but of course bringing new features to the gallery now takes more than a week...
